templates/workspaces/base/terraform/network/locals.tf

```diff
@@ -6,4 +6,8 @@ locals {
   vnet_subnets                   = cidrsubnets(var.address_space, 1, 1)
   services_subnet_address_prefix = local.vnet_subnets[0]
   webapps_subnet_address_prefix  = local.vnet_subnets[1]
+  tre_workspace_tags = {
+    tre_id           = var.tre_id
+    tre_workspace_id = var.tre_resource_id
+  }
```

Comment:
Same as above - pass in the map **tre_workspace_tags** as a variable instead of recreating it in this module.